test/sde/sde_dynamical.jl

```diff
@@ -19,6 +19,15 @@ g(u,p,t) = 1
     sim1  = analyticless_test_convergence(dts,prob1,BAOAB(gamma=γ),(1/2)^10;trajectories=Int(2e2),use_noise_grid=false)
     display(sim1.𝒪est)
     @test abs(sim1.𝒪est[:weak_final]-1) < 0.3
+
+    sim1  = analyticless_test_convergence(dts,prob1,ABOBA(gamma=γ),(1/2)^10;trajectories=Int(5e2),use_noise_grid=false)
+    display(sim1.𝒪est)
+    @test abs(sim1.𝒪est[:weak_final]-1) < 0.3
```

Comment:
Weak final order 1 seems strange to me -- aren't these methods supposed to be second-order in weak convergence? 

(Having said this, based on the paper https://arxiv.org/abs/1304.3269, it seems that the error may increase when you lower the step size (Figure 3) and second-order convergence can only be seen over very small intervals (Figure 4), so I am not too worried but maybe we should still try to reproduce the plots?)

Reply:
I agree that they should be 2nd order, but the numerics give me first order. I will try to reproduce the figure 4 in the paper over this smaller step size range to see what it give.

Reply:
Do you have an update for this plot?

Reply:
![fig3](https://github.com/SciML/StochasticDiffEq.jl/assets/62566610/39fbad80-5008-4fd1-a23b-338c0a9ef714)
Here is already the figure 3 of the paper (OBABO here correspond to BP there). Figure 4 is running, I should get it by tomorrow

Reply:
There is only 200 trajectories of 10^7 timesteps so less trajectories than the paper, but results are similar

Reply:
[fig4.pdf](https://github.com/SciML/StochasticDiffEq.jl/files/12854717/fig4.pdf)
And now the panel (a) and (c) of the figure 4 of the paper, I cannot go as far in stepsize since SciML detect instability at timestep starting from 0.24 and abort the computation so I went to smaller timestep to better see the scaling. There is less trajectories than in the paper, here 1000 of 10^7 timestep to be compared to 500 trajectories of 10^9 timestep, but the computation is quite demanding so I went for less timestep. The main features of the figure 4 are there (with BP=OBABO)